### PR TITLE
Cards E2E FOAM Merge

### PR DESCRIPTION
**Pull request by tharmaman**
- Application popup fixes
- Overall UI touchups
- Fix login
- Fix double title bug
- Generic widget touchups
- Schedulable name factory

### CODE REVIEW

File: src/foam/comics/v2/DAOBrowseControllerView.js [hunk 1]
```diff
@@ -297,15 +303,15 @@ foam.CLASS({
...
-                })
+                // .callIf(config$browseViews.length > 1 , function() {
+                //   this
+                //     .start(self.IconChoiceView, {
+                //       choices:config$browseViews.map(o => [o.view, o.icon]),
+                //       data$: self.browseView$
+                //     })
+                //       .addClass(self.myClass('altview-container'))
+                //     .end();
+                // })
```
Comment (sarthak-marwaha) [inline src/foam/comics/v2/DAOBrowseControllerView.js]:
```suggestion
```

File: src/foam/u2/view/IconChoiceView.js [hunk 2]
```diff
@@ -76,14 +115,17 @@ foam.CLASS({
...
-            start('span').
-              tag({
+            start({
                 class: 'foam.u2.tag.Image',
-                data: c[1]
+                data: c[1],
+                embedSVG: true
               }).
-              enableClass(this.myClass('disabled-icon'), self.slot(function(data) { return data === c[0] })).
+              // enableClass(this.myClass('disabled-icon'), self.slot(function(data) { return data === c[0] })).
```
Comment (sarthak-marwaha) [inline src/foam/u2/view/IconChoiceView.js]:
```suggestion
```

File: src/foam/u2/view/IconChoiceView.js [hunk 3]
```diff
@@ -29,12 +29,51 @@ foam.CLASS({
...
       line-height: 48px;
-      margin-left: 16px;
     }
 
 
-    ^disabled-icon {
-      opacity: 0.2;
+    // ^disabled-icon {
+    //   opacity: 0.2;
+    // }
```
Comment (sarthak-marwaha) [inline src/foam/u2/view/IconChoiceView.js]:
```suggestion
```

File: src/foam/u2/dialog/ApplicationPopup.js [hunk 4]
```diff
@@ -217,8 +264,9 @@ foam.CLASS({
             .start()
               .addClass(this.myClass('header-center'))
               .start(this.Image, {
-                data$: this.slot(function(theme$topNavLogo) {
-                  return theme$topNavLogo;
+                data$: this.slot(function(theme$loginImage) {
+                  // Temp change for demo, create ImageTokens before merging
+                  return theme$loginImage;
```
Comment (sarthak-marwaha) [inline src/foam/u2/dialog/ApplicationPopup.js]: Can we revert this for now

File: src/foam/u2/wizard/wizardlet/WizardletSection.js [hunk 5]
```diff
@@ -133,10 +133,12 @@ foam.CLASS({
       if ( ! opt_spec ) opt_spec = {};
       var ctx = this.wizardlet.__subSubContext__.createSubContext({ wizardController: this.wizardlet.wizardController });
 
-      this.analyticsAgent.pub('event', {
-        name: 'VIEW_LOAD_' + this.wizardlet.id,
-        extra: foam.json.stringify(this.customView)
-      });
+      if ( this.analyticsAgent ){
+        this.analyticsAgent.pub('event', {
```
Comment (sarthak-marwaha) [inline src/foam/u2/wizard/wizardlet/WizardletSection.js]: this should be 
```suggestion
        ctx.analyticsAgent.pub('event', {
```

File: src/foam/u2/dialog/ApplicationPopup.js (hunk 4 above)
Comment (tharmaman) [reply src/foam/u2/dialog/ApplicationPopup.js]: done

**Pull request merged**